docs/miscellaneous.rst

```diff
@@ -58,6 +61,51 @@ So for the following contract snippet::
 
 The position of ``data[4][9].b`` is at ``keccak256(uint256(9) . keccak256(uint256(4) . uint256(1))) + 1``.
 
+Adding elements to or removing them from a dynamically-sized array will increment or decrement the number
```

Comment:
I don't really understand this paragraph. Elements can be stored in the same slot.